install/pocketbase-install.sh

```diff
@@ -51,4 +52,5 @@ customize
 msg_info "Cleaning up"
 $STD apt-get -y autoremove
 $STD apt-get -y autoclean
+rm -rf /tmp/pocketbase.zip
```

Comment:
move above `autoremove`

Reply:
I'm confused. I checked a few other scripts and they all did it _after_ autoremove.

Reply:
fixed.